texastribune/donations-app
# A few updates

#### What's this PR do?

Adds a few small updates, including updated level benefits, updated renew button copy, updated mobile content, and a fix of dollar signs missing from circle amounts.
#### Why are we doing this? How does it help us?

To improve how everything looks on mobile, have our most updated benefits, and fix missing dollar signs.
#### How should this be manually tested?

Visit the page locally, and confirm dollar signs are on circle amounts, mobile looks good (tagline shouldn't be showing on 'Membership Levels' page, renew button now says 'Renew Today', and updated video height). Confirm tweeting works as expected.
#### Are there performance implications? If adding JS can it be done async? Do images/CSS/JS have cache headers set?

n/a
#### What are the relevant tickets?

Part of donations app Sprint task.
#### Next steps?

Keep improving.
#### Smells?
#### TODOs:
#### Has the relevant documentation/wiki been updated?
#### Technical debt note
